device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/hwsku.json

```diff
@@ -0,0 +1,160 @@
+            "port_type": "RJ45"
+        },
+        "Ethernet8": {
+            "port_type": "RJ45"
+        },
+        "Ethernet9": {
+            "port_type": "RJ45"
+        },
+        "Ethernet10": {
+            "port_type": "RJ45"
+        },
+        "Ethernet11": {
+            "port_type": "RJ45"
+        },
+        "Ethernet12": {
+            "port_type": "RJ45"
+        },
+        "Ethernet13": {
+            "port_type": "RJ45"
+        },
+        "Ethernet14": {
+            "port_type": "RJ45"
+        },
+        "Ethernet15": {
+            "port_type": "RJ45"
+        },
+        "Ethernet16": {
+            "port_type": "RJ45"
+        },
+        "Ethernet17": {
+            "port_type": "RJ45"
+        },
+        "Ethernet18": {
+            "port_type": "RJ45"
+        },
+        "Ethernet19": {
+            "port_type": "RJ45"
+        },
+        "Ethernet20": {
+            "port_type": "RJ45"
+        },
+        "Ethernet21": {
+            "port_type": "RJ45"
+        },
+        "Ethernet22": {
+            "port_type": "RJ45"
+        },
+        "Ethernet23": {
+            "port_type": "RJ45"
+        },
+        "Ethernet24": {
+            "port_type": "RJ45"
+        },
+        "Ethernet25": {
+            "port_type": "RJ45"
+        },
+        "Ethernet26": {
+            "port_type": "RJ45"
+        },
+        "Ethernet27": {
+            "port_type": "RJ45"
+        },
+        "Ethernet28": {
+            "port_type": "RJ45"
+        },
+        "Ethernet29": {
+            "port_type": "RJ45"
+        },
+        "Ethernet30": {
+            "port_type": "RJ45"
+        },
+        "Ethernet31": {
+            "port_type": "RJ45"
+        },
+        "Ethernet32": {
+            "port_type": "RJ45"
+        },
+	"Ethernet33": {
+            "port_type": "RJ45"
+        },
+	"Ethernet34": {
+            "port_type": "RJ45"
+        },
+	"Ethernet35": {
+            "port_type": "RJ45"
+        },
+	"Ethernet36": {
+            "port_type": "RJ45"
+        },
+	"Ethernet37": {
+            "port_type": "RJ45"
+        },
+	"Ethernet38": {
+            "port_type": "RJ45"
+        },
+	"Ethernet39": {
+            "port_type": "RJ45"
+        },
+	"Ethernet40": {
+            "port_type": "RJ45"
+        },
+	"Ethernet41": {
+            "port_type": "RJ45"
+        },
+	"Ethernet42": {
+            "port_type": "RJ45"
+        },
+	"Ethernet43": {
+            "port_type": "RJ45"
+        },
+	"Ethernet44": {
+            "port_type": "RJ45"
+        },
+	"Ethernet45": {
+            "port_type": "RJ45"
+        },
+	"Ethernet46": {
+            "port_type": "RJ45"
+        },
+	"Ethernet47": {
+            "port_type": "RJ45"
+        },
+	"Ethernet48": {
+            "port_type": "SFP+"
+        },
+	"Ethernet49": {
+            "port_type": "SFP+"
+        },
+	"Ethernet50": {
+            "port_type": "SFP+"
+        },
+	"Ethernet51": {
+            "port_type": "SFP+"
+        }
+    }	
+}
```

Comment:
Add newline at EOF

Reply:
Joe - added the newline at EOF - can you please take another look - thanks